From 13d0e8ea85fbbc4d53709a0fb28ebd66876884b4 Mon Sep 17 00:00:00 2001
From: Alexandre Derumier <aderumier@odiso.com>
Date: Mon, 31 Dec 2018 09:50:06 +0100
Subject: [PATCH] enable bgp daemon

---
 tools/etc/frr/daemons | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/etc/frr/daemons b/tools/etc/frr/daemons
index 9077606..2d11d6a 100644
--- a/tools/etc/frr/daemons
+++ b/tools/etc/frr/daemons
@@ -14,7 +14,7 @@
 #
 # The watchfrr and zebra daemons are always started.
 #
-bgpd=no
+bgpd=yes
 ospfd=no
 ospf6d=no
 ripd=no
-- 
2.11.0
