From f098e90f4d690e771f63f48e0540a470faa91892 Mon Sep 17 00:00:00 2001
From: Alexandre Derumier <aderumier@odiso.com>
Date: Wed, 25 Nov 2020 12:41:32 +0100
Subject: [PATCH] enable bgp && bfd daemons

Signed-off-by: Alexandre Derumier <aderumier@odiso.com>
---
 tools/etc/frr/daemons | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/etc/frr/daemons b/tools/etc/frr/daemons
index b1526888e..27bc120a7 100644
--- a/tools/etc/frr/daemons
+++ b/tools/etc/frr/daemons
@@ -14,7 +14,7 @@
 #
 # The watchfrr, zebra and staticd daemons are always started.
 #
-bgpd=no
+bgpd=yes
 ospfd=no
 ospf6d=no
 ripd=no
@@ -27,7 +27,7 @@ eigrpd=no
 babeld=no
 sharpd=no
 pbrd=no
-bfdd=no
+bfdd=yes
 fabricd=no
 vrrpd=no
 pathd=no
-- 
2.30.2

